src/utils/utils_part.h

```diff
@@ -50,6 +51,51 @@ static inline ocf_part_id_t ocf_part_class2id(ocf_cache_t cache, uint64_t class)
 	return PARTITION_DEFAULT;
 }
 
+static inline uint32_t ocf_part_get_occupancy(ocf_cache_t cache,
+		ocf_part_id_t part_id)
+{
+	return cache->user_parts[part_id].runtime->curr_size;
+}
+
+static inline uint32_t ocf_part_get_min_size(ocf_cache_t cache,
+		ocf_part_id_t part_id)
+{
+	uint64_t ioclass_size;
+
+	if (unlikely(cache->user_parts[part_id].config->min_size > 100)) {
+		ocf_cache_log(cache, log_err, "Ioclass %hu: invalid min size %u\n",
+				part_id, cache->user_parts[part_id].config->min_size);
+		ENV_BUG();
+	}
```

Comment:
I would expect to see this check wherever we set the min value rather than here in get_min_size()